CHANGELOG.md

```diff
@@ -246,60 +324,114 @@
+* Allow skin scaling from preferences [#3960](https://github.com/mixxxdj/mixxx/pull/3960)
+* Invert scroll wheel waveform zoom direction to mach other applications [#4195](https://github.com/mixxxdj/mixxx/pull/4195)
 * Fix crash if no skin is available
   [#3918](https://github.com/mixxxdj/mixxx/pull/3918)
   [#3939](https://github.com/mixxxdj/mixxx/pull/3939)
-* Inverted scroll wheel waveform zoom direction to mach other applications [#4195](https://github.com/mixxxdj/mixxx/pull/4195)
 * Fix leaked controls [#4213](https://github.com/mixxxdj/mixxx/pull/4213) [lp:1912129](https://bugs.launchpad.net/mixxx/+bug/1912129)
+* Shade: remove initial setting of now accessible effect controls [#4398](https://github.com/mixxxdj/mixxx/pull/4398) [lp:1946811](https://bugs.launchpad.net/mixxx/+bug/1946811)
+* Fix switching from Shade to other skins [#4421](https://github.com/mixxxdj/mixxx/pull/4421) [lp:1946812](https://bugs.launchpad.net/mixxx/+bug/1946812)
+* Use double click to reset knobs and sliders [#4509](https://github.com/mixxxdj/mixxx/pull/4509) [lp:1875999](https://bugs.launchpad.net/mixxx/+bug/1875999)
+* Use info not warning for skin COs [#4525](https://github.com/mixxxdj/mixxx/pull/4525)
+
+### Effects
+
+* Add a Noise effect [#2921](https://github.com/mixxxdj/mixxx/pull/2921)
+* Use '---' instead of 'None' for empty slots to spot them easier [#4469](https://github.com/mixxxdj/mixxx/pull/4469)
+* Effect refactoring: Effect chain saving/loading, parameter hiding/rearrangement, effect preferences overhaul
+  [#4467](https://github.com/mixxxdj/mixxx/pull/4467)
+  [#4431](https://github.com/mixxxdj/mixxx/pull/4431)
+  [#4426](https://github.com/mixxxdj/mixxx/pull/4426)
+  [#4457](https://github.com/mixxxdj/mixxx/pull/4457)
+  [#4456](https://github.com/mixxxdj/mixxx/pull/4456)
+  [#4459](https://github.com/mixxxdj/mixxx/pull/4459)
+  [#4462](https://github.com/mixxxdj/mixxx/pull/4462)
+  [#4466](https://github.com/mixxxdj/mixxx/pull/4466)
+  [#4468](https://github.com/mixxxdj/mixxx/pull/4468)
+  [#4472](https://github.com/mixxxdj/mixxx/pull/4472)
+  [#4470](https://github.com/mixxxdj/mixxx/pull/4470)
+  [#4471](https://github.com/mixxxdj/mixxx/pull/4471)
+  [#4483](https://github.com/mixxxdj/mixxx/pull/4483)
+  [#4482](https://github.com/mixxxdj/mixxx/pull/4482)
+  [#4484](https://github.com/mixxxdj/mixxx/pull/4484)
+  [#4486](https://github.com/mixxxdj/mixxx/pull/4486)
+  [#4502](https://github.com/mixxxdj/mixxx/pull/4502)
+  [#4501](https://github.com/mixxxdj/mixxx/pull/4501)
+  [#4518](https://github.com/mixxxdj/mixxx/pull/4518)
+  [#4532](https://github.com/mixxxdj/mixxx/pull/4532)
+  [#4461](https://github.com/mixxxdj/mixxx/pull/4461)
+  [#4548](https://github.com/mixxxdj/mixxx/pull/4548)
+  [#4503](https://github.com/mixxxdj/mixxx/pull/4503)
 
 ### Other
 
-* Improve/fix the build system
-  [#2796](https://github.com/mixxxdj/mixxx/pull/2796)
-  [#2937](https://github.com/mixxxdj/mixxx/pull/2937)
+* Improve/fix the CMake build system
```

Comment:
Unless you explain what was fixed/improved, this does not make sense to mention IMHO.